### ISSUE TITLE
Malicious beatmap file can replace intro sequence music

### ISSUE THREAD

**Issue by CenTdemeern1** [ppy/osu#17659]
### Type

Other

### Bug description

TL;DR: The end-user can override the default intro sequence beatmaps by importing new ones.

By taking one of the default intro sequence beatmaps from [here](https://github.com/ppy/osu-resources/tree/master/osu.Game.Resources/Tracks), replacing the audio file with a tool such as 7zip, and importing it, you can replace the default entry, replacing the intro music when you reboot the game and making the beatmap visible in the beatmap select.

This can be undone by deleting the beatmap and restarting the game, since a missing intro sequence beatmap on bootup imports the default.

### Screenshots or videos

https://youtu.be/bRqytvZ5vIM

### Version

2022.327.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/8417669/database.log)
[performance.log](https://github.com/ppy/osu/files/8417671/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8417672/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8417673/updater.log)

**Comment by CenTdemeern1** [comment]
Oh, before I forget, here is an example of such a malicious beatmap file. (You will need to rename this to an .osz file.)
It replaces triangles.
[beatmap.zip](https://github.com/ppy/osu/files/8417692/beatmap.zip)
You can easily make one for circles and welcome to osu! as well if you follow the instructions I gave.
